### ISSUE TITLE
Show version number in about dialog

### ISSUE THREAD

**Issue by heldderarbeit** [kra-mo/sly#28]
The About dialog is currently not showing the version number.

![Screenshot From 2024-09-21 05-09-45](https://github.com/user-attachments/assets/e92c14a5-a5cc-40a2-b624-7348307b5d71)

It would be good to have it for bug reporting purposes, to not have to rely on `flatpak list | grep Sly.` Most Gnome / Kde apps show the version number in the about dialog.

![Screenshot From 2024-09-21 05-08-52](https://github.com/user-attachments/assets/7e6f58ab-4a07-45ab-b6d6-b188ab44dec1)

![Screenshot From 2024-09-21 05-07-13](https://github.com/user-attachments/assets/c7f9bc88-2793-4470-bf91-2ac7140a34df)

**Comment by kra-mo** [comment]
I agree, the issue with that is there is no way to easily programmatically get the current version and if I rely on hard-coding it, I will probably forget to update it 😅

**Comment by kra-mo** [comment]
Plus I plan to only officially support installations from auto-updating software stores, so outdated versions shouldn't be that big of an issue.

**Comment by kra-mo** [comment]
Heads up: Sly has been migrated to Codeberg. This discussion continues [here](https://codeberg.org/kramo/Sly/issues/28).
